tests/src/JIT/Performance/CodeQuality/Benchstones/BenchF/BenchMk2/BenchMk2.cs

```diff
@@ -41,6 +41,27 @@ private static bool Bench()
         return true;
     }
 
+    private static bool VerifyResult()
+    {
+        const float DIFF = 0.0000001f;
+        bool result = Math.Abs(s_p - 3.1415927f) < DIFF;
+        result &= s_n == Iterations;
+
+#if DEBUG
```

Comment:
I don't particularly like us testing different results in debug vs release....

Reply:
Yeah, it's using the iteration number to actually calculate the results though. I'm not sure why (as per the comment below).